util/junitxml/junitxml.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 Kushal Shukla
```

Comment:
Please use
```suggestion
// Copyright 2024 The Prometheus Authors
```

Note: using individual name was the reason we rejected the original PR.